mic/src/mic/VassalXWSPilotPieces.java

```diff
@@ -241,8 +241,19 @@ private String acronymizer(String cardName, boolean isUnique, boolean shortName)
         String[] words = name.split(" ");
 
         if (name.length() <= maxLength) return name;
-        if (words.length == 1) return name.substring(0, maxLength);
-        if (isUnique && words[0].length() <= maxLength) return words[0];
+        if (words.length == 1) {
+            return name.substring(0, maxLength);
+        } else if (words.length > 1 && isUnique) {
```

Comment:
`words.length > 1` is unnecessary. It will always be `true`, as the previous condition already returns if `words.length == 1`